code/modules/vehicles/ridden.dm

```diff
@@ -60,6 +60,11 @@
 	if(key_type && !is_key(inserted_key))
 		to_chat(user, "<span class='warning'>[src] has no key inserted!</span>")
 		return FALSE
+	if(canmove && (user.get_num_arms() < arms_required))
+		to_chat(user, "<span class='warning'>You don't have enough legs to operate the pedals!</span>")

```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>You don't have enough arms to operate the handlebars!</span>")
```
Otherwise misleading as the number of available arms is checked instead.

Reply:
Oops that was supposed to check legs not arms.